README.md

```diff
@@ -1,20 +1,38 @@
-# NixOS on BPi-R3 Example
+# NixOS on BPi-R3
 
-This is an example of booting NixOS on a BPi-R3.
+This respository contains community supported patches that are necessary to run NixOS on BananaPi-R3 and use it as a fully fledget router.
+
+## Getting started
 
 Build an SD-Image with:
 
 ```
 $ nix build -L '.#nixosConfigurations.bpir3.config.system.build.sdImage'
 ```
 
+## Features
```

Comment:
maybe a table would be suitable here so we could distinguish between various levels of availability for these features:

* hardware support
* kernel support
* configuration support demonstrates via this repo


WDYT?

Reply:
I see your point but do we want to keep items in the table that are not applicable to bpir3 hardwarde? Because if not, then there is no point in having such a distinction. 

I agree regarding the distinction between possible to configure and supported by the flake.

Reply:
I rewrote it as a table but I am not 100% happy with it. For example, not sure how vlans+wifi would fit into that. 

Any idea how to improve it?